src/sentry/migrations/0372_backfill_perf_issue_events_issue_platform.py

```diff
@@ -0,0 +1,216 @@
+from sentry.utils.performance_issues.performance_detection import EventPerformanceProblem
+
+
+def backfill_eventstream(apps, schema_editor):
+    """
+    Backfills Performance-issue events from the transaction table to the IssuePlatform dataset(search_issues).
+    1. Read all transactions that back a performance-issue(s) with the following criteria:
+        a. non-null group_ids column
+        b.
+    2. Prepare and format each transaction to be submitted through the eventstream.
+    3. Send it through the eventstream.
+    """
+
+    # TODO: need to make sure event_type is correct in eventstream.send()
+
+    Group = apps.get_model("sentry", "Group")
+    GroupHash = apps.get_model("sentry", "GroupHash")
+
+    project_perf_issues = (
+        Group.objects.all()
+        .filter(
+            type__in=(
+                PerformanceSlowDBQueryGroupType.type_id,
+                PerformanceRenderBlockingAssetSpanGroupType.type_id,
+                PerformanceNPlusOneGroupType.type_id,
+                PerformanceConsecutiveDBQueriesGroupType.type_id,
+                PerformanceFileIOMainThreadGroupType.type_id,
+                PerformanceNPlusOneAPICallsGroupType.type_id,
+                PerformanceMNPlusOneDBQueriesGroupType.type_id,
+                PerformanceUncompressedAssetsGroupType.type_id,
+            )
+        )
+        .values("id", "project_id")
+        .annotate(issue_counts=Count("id"))
+        .order_by("project_id")
+    )
+
+    for project_perf_issue in project_perf_issues:
+        backfill_by_project(project_perf_issue["project_id"], Group, GroupHash)
+
+
+def backfill_by_project(project_id: int, Group: Any, GroupHash: Any):
+    # retrieve rows
+    rows = _query_performance_issue_events(
+        project_ids=[project_id],
+        start=datetime(2008, 5, 8),
+        end=datetime.now(),
+    )
+
+    for row in rows:
+        try:
+            # don't need to store in node_store since it should be there already
+            # create issue occurrence
+            # save issue occurrence
+            # event, occurrence_data, group_id = _map_performance_issue_row_to_params(row)
+            project_id = row["project_id"]
+            group_id = row["group_id"]
+            event_id = row["event_id"]
+
+            group: Group = Group.objects.get(id=group_id)
+            group_hash: GroupHash = GroupHash.objects.get(group_id=group_id)
+
+            event: Event = lookup_event(project_id=project_id, event_id=event_id)
+
+            # need to map the base raw data to an issue occurrence
+            # make sure this is consistent with how we plan to ingest performance issue occurrences
+            occurrence_data: IssueOccurrenceData = IssueOccurrenceData(
+                id=uuid.uuid4().hex,  # TODO: need to generate this
+                project_id=project_id,
+                event_id=event_id,
+                fingerprint=[group_hash.hash],
+                issue_title="",
+                subtitle="",
```

Comment:
`issue_title` can be `group.title`. This should be correct at least, probably confirm to be sure.

I think that `subtitle` would be `group.culprit`, also worth validating.